packages/ui5-cc-spreadsheetimporter/src/controller/Util.ts

```diff
@@ -181,16 +181,21 @@ export default class Util extends ManagedObject {
 		URL.revokeObjectURL(url);
 	}
 
-	static async loadUI5RessourceAsync(libraryName: string): Promise<any> {
+	static async loadUI5RessourceAsync(moduleName: string): Promise<any> {
+		const alreadyLoadedLibrary = sap.ui.require(moduleName);
+		if (alreadyLoadedLibrary) {
+			return Promise.resolve(alreadyLoadedLibrary);
+		}
```

Comment:
Now that I see it in a PR. I'm stupid for not also renaming this to `alreadyLoadedModule`. Written when it still had `libraryName` in the params. ^^